apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -163,6 +168,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
         render={({ field: { value, onChange } }) => (
           <SettingsToggle
             title={t("disable_notes")}
+            {...shouldLockDisableProps("hideCalendarNotes")}
```

Comment:
This function returns `disabled` and `isLocked` properties for a given field name, `SettingsToggle ` now supports `disabled` and `isLocked` to be able to show an indicator in the label, indicator that comes in the isLocked flag itself.